### PR TITLE
Fix `GetCDNAuthToken` after Steam change

### PR DESCRIPTION
**Pull request by oureveryday**
Move the GetCDNAuthToken from SteamApps to SteamContent

### CODE REVIEW

**Comment by xPaw** [comment]
Result code still needs to be checked/available

**Comment by oureveryday** [comment]
I didn't see result code in CContentServerDirectory_GetCDNAuthToken_Response

**Comment by xPaw** [comment]
Service response itself should have a result. You should try testing it on invalid hostname / depotid, I would expect that to give a bad eresult.

File: SteamKit2/SteamKit2/Steam/Handlers/SteamApps/Callbacks.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```
Comment (xPaw) [inline SteamKit2/SteamKit2/Steam/Handlers/SteamApps/Callbacks.cs:1]: can you undo these changes please (utf BOM probably)

Comment (oureveryday) [reply SteamKit2/SteamKit2/Steam/Handlers/SteamApps/Callbacks.cs]: Fixed that

**Comment by oureveryday** [comment]
> Service response itself should have a result. You should try testing it on invalid hostname / depotid, I would expect that to give a bad eresult.

If a invalid hostname / depotid passed, it will return null result.

**Comment by xPaw** [comment]
> it will return null result.

Yeah but what's the `message.Result`?

**Comment by oureveryday** [comment]
> Yeah but what's the `message.Result`?

Don't know where it is...

**Comment by oureveryday** [comment]
Fixed that.

**Comment by xPaw** [comment]
xmldoc in cdn client needs updating, see the failing build
